controllers/articles.controllers.js

```diff
@@ -30,14 +31,8 @@ exports.getArticles = (req, res, next) => {
 exports.getCommentsForArticle = (req, res, next) => {
   const { article_id } = req.params;
 
-  const fetchCommentQuery = fetchCommentsForArticle(article_id);
-  const checkIdExists = checkArticleIdExists(article_id);
-
-  const queries = [fetchCommentQuery, checkIdExists];
-
-  Promise.all(queries)
-    .then((response) => {
-      comments = response[0];
```

Comment:
checkIdExists now invoked in artical.model. ref to https://github.com/deanlicious/nc-news/pull/8#pullrequestreview-1826849174